discord_bots/tasks.py

```diff
@@ -49,6 +52,150 @@
 
 _log = logging.getLogger(__name__)
 
+async def add_players(session: sqlalchemy.orm.Session):
```

Comment:
did these just get moved around from after the merge?

Reply:
Meant to add this in the comments.  I just manually alphabetized them because the file was starting to get a bit long